projects/travel-tracker.md

```diff
@@ -26,14 +27,28 @@ Tuesday, Week 6 - Project due at 12PM Mountain Time.
 
 # Requirements
 
+## Workflow
+You will want to submit PRs to your accountabilibuddy to:
+
+* You must give your accountabilibuddy collaboration access to your repo.
+* You must submit *at least* 2 PRs to your accountabilibuddy for review.
+* You must wait for your accountabilibuddy to review your PRs, and allow THEM to merge any PRs you submit.
```

Comment:
I don't think this should be a requirement since students are on a tight deadline. Their work getting merged and being able to move on shouldn't be dependent on another student.

Reply:
Sounds great to update Overlook as well! I can go ahead and update for this inning, no worries.